packages/py-api/py_api/functionality/hackathon/teams/teams_utility_functions.py

```diff
@@ -73,6 +76,19 @@ def fetch_team(
 
         return HackathonTeam(**team)
 
+    @classmethod
+    def fetch_teams_by_condition(cls, conditions: Dict[str, Any]) -> List[HackathonTeam]:
+        # if conditions are empty it will return all the teams
+        filtered_teams = list(t_col.find(conditions))
+        return [HackathonTeam(**team) for team in filtered_teams]
+
+    @classmethod
+    def team_has_reach_max_cap(cls, team: HackathonTeam) -> bool:
+        if (len(team.team_members) < 6):
```

Comment:
return len(team.team_members) < 6

Reply:
or for better readability in the other function, rename to team_has_available_space(..)


return len(members) < 6


this will allow you to do:

if available_team := team_has_available_space(team):

which is more readable than:

if not available_team := TeamsUtilities.team_has_reach_max_cap(team):